Rakefile

```diff
@@ -104,3 +104,9 @@ task :release, :version do |task, args|
                    name: version,
                    body: changelog.split(/^# /)[2].strip)
 end
+
+desc 'Runs pod install on the Demo project, used for testing too.'
+task :pod_install do
+  sh "brew install xcproj" unless `which xcproj`.strip.length > 0
+  sh "cd Demo ; COCOAPODS_DISABLE_DETERMINISTIC_UUIDS=true bundle exec pod install"
```

Comment:
Cool, I'll remove it but I'm going to wait for 🍏 to push the commit, so I don't have to wait for CI twice.